apis/database/v1beta1/rdsinstance_types.go

```diff
@@ -1154,9 +1154,10 @@ type RDSInstanceStatus struct {
 // An RDSInstance is a managed resource that represents an AWS Relational
 // Database Service instance.
 // +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.bindingPhase"
-// +kubebuilder:printcolumn:name="STATE",type="string",JSONPath=".status.state"
+// +kubebuilder:printcolumn:name="STATE",type="string",JSONPath=".status.atProvider.dbInstanceStatus"
 // +kubebuilder:printcolumn:name="CLASS",type="string",JSONPath=".spec.classRef.name"
-// +kubebuilder:printcolumn:name="VERSION",type="string",JSONPath=".spec.engineVersion"
+// +kubebuilder:printcolumn:name="VERSION",type="string",JSONPath=".spec.forProvider.engine"
```

Comment:
there's 2 columns in a row that are called `VERSION`, is that intentional?

Reply:
No, it's my developer eyes turning blind to things that IDE doesn't highlight. Good catch!